src/manage_sql.c

```diff
@@ -32117,6 +32117,20 @@ target_reverse_lookup_unify (target_t target)
                      " WHERE id = %llu;", target);
 }
 
+/**
+ * @brief Return the allow_simult_ips_same_host value of a target.
+ *
+ * @param[in]  target  Target.
+ *
+ * @return The allow_simult_ips_same_host value if available, else NULL.
+ */
+char*
+target_allow_simult_ips_same_host (target_t target)
```

Comment:
Not a big deal, but wouldn't returning an int be cleaner?  It's an integer column in the db, and being compared to 0 by the caller.

Reply:
I've considered using int but then I've done it this way mainly for consistency with the similar target options reverse_lookup_only and reverse_lookup_unify. Perhaps we should change all of them in a separate PR.